src/Kuzzle.ts

```diff
@@ -164,6 +169,11 @@ export class Kuzzle extends KuzzleEventEmitter {
        * If set to `auto`, the `autoQueue` and `autoReplay` are also set to `true`
        */
       offlineMode?: 'auto';
+      /**
+       * Show deprecation warning in development mode (hidden either way in production)
+       * Default: `true`
+       */
+      deprecationWarning?: boolean;
```

Comment:
Shouldn't you put a default value here?

Reply:
Well the default value doesn't seems to be defined here when looking at the others parameters but rather just after (from line 202)